rocAL/rocAL/include/api/rocal_api_data_loaders.h

```diff
@@ -707,11 +713,13 @@ extern "C"  RocalImage  ROCAL_API_CALL rocalJpegCaffe2LMDBRecordSourcePartialSin
                                                             unsigned shard_id,
                                                             unsigned shard_count,
                                                             bool is_output,
+                                                            std::vector<double>& area_factor, 
+                                                            std::vector<double>& aspect_ratio, 
+                                                            unsigned num_attempts,
                                                             bool shuffle = false,
                                                             bool loop = false,
-                                                            RocalImageSizeEvaluationPolicy decode_size_policy = ROCAL_USE_MAX_SIZE,
+                                                            RocalImageSizeEvaluationPolicy decode_size_policy = ROCAL_USE_USER_GIVEN_SIZE_RESTRICTED,
```

Comment:
please adjust spacing here

Reply:
I am not seeing any spacing issues here in github / my editor